packaging/ondemand.spec

```diff
@@ -227,12 +229,12 @@ EOF
 EOS
 
 %post
-if [ "$1" -eq 2 ]; then
 %if %{with scl_apache}
+if [ "$1" -eq 2 ]; then
```

Comment:
Was this like a cherry-pick from master or something? This change was made a while ago.

Reply:
same as other accidental spec commit.